fix(curriculum): add missing tested requirements to travel agency lab (#68672)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/blocks/lab-travel-agency-page/669e2f60e83c011754f711f9.md

1. You should have an `html` element with `lang` set to `en`.
1. You should have a `head` element containing a `meta` void element with `charset` set to `utf-8` and a `title` with the text `Travel Agency Page`.
1. You should have a `meta` tag in your `head` element that contains a short description of your website for SEO.
1. You should have an `h1` element to present your travel destinations. It should be the only `h1` element on your page.
1. You should have a paragraph below the `h1` element introducing the travel opportunities.
1. You should have an `h2` element with the text `Packages`.
1. You should have a `p` element introducing briefly the various packages.
1. You should have an unordered list element with two list items. The two list items should have the text `Group Travels` and `Private Tours`, respectively. The text of each list item should be enclosed by an anchor element.
1. You should have an `h2` element with the text `Top Itineraries`.

> 1. You should have at least three `figure` elements, each containing an anchor element and a `figcaption` element. Each `figcaption` should contain some text.

Reviewer: ```suggestion
1. You should have at least three `figure` elements, each containing an anchor element and a `figcaption` element with a description of the image (see next user story).
```

> 1. You should have at least three `figure` elements, each containing an anchor element and a `figcaption` element with a description of the image (see next user story).

Reviewer: ```suggestion
1. You should have at least three `figure` elements, each containing an anchor element and a `figcaption` element that describes the figure's image.
```
